File: 2265-partition-array-according-to-given-pivot/partition-array-according-to-given-pivot.py
class Solution:
    def pivotArray(self, nums: List[int], p: int) -> List[int]:
        a,b,c=[],[],[]
        for i in nums:
            if(i<p):
                a.append(i)
            elif(i==p):
                b.append(i)
            else:
                c.append(i)
        return a+b+c

File: 2265-partition-array-according-to-given-pivot/partition-array-according-to-given-pivot.cpp
class Solution {
public:
    vector<int> pivotArray(vector<int>& nums, int p) {
        vector<int>a,b,c;
        for(auto it:nums){
            if(it<p){
                a.push_back(it);
            }
            else if(it==p){
                b.push_back(it);
            }
            else{
                c.push_back(it);
            }
            
        }
        vector<int>ans;
            for(auto it:a){
                ans.push_back(it);
            }
            for(auto it:b){
                ans.push_back(it);
            }
            for(auto it:c){
                ans.push_back(it);
            }
            return ans;
    }
};